src/main/java/com/GeorgesServer/app/com/GeorgesServer/handler/DefaultHandler.java

```diff
@@ -1,20 +1,21 @@
 package com.GeorgesServer.app.com.GeorgesServer.handler;
 
 
-import com.GeorgesServer.app.ClientRequest;
-import com.GeorgesServer.app.HttpResponseBuilder;
-import com.GeorgesServer.app.ServerResponse;
+import com.GeorgesServer.app.com.GeorgesServer.request.ClientRequest;
+import com.GeorgesServer.app.com.GeorgesServer.response.HttpResponseBuilder;
+import com.GeorgesServer.app.com.GeorgesServer.response.ServerResponse;
 
-public class RequestHandler implements IHandler {
+public class DefaultHandler implements IHandler {
 
     private HttpResponseBuilder responseBuilder;
 
-    public RequestHandler(HttpResponseBuilder responseBuilder) {
+    public DefaultHandler(HttpResponseBuilder responseBuilder) {
         this.responseBuilder = responseBuilder;
     }
 
     public ServerResponse handle(ClientRequest clientRequest) {
 
+
         responseBuilder.buildHttpVersion();
         responseBuilder.buildOkStatus();
         return responseBuilder.getResponse();
```

Comment:
I'm assuming this PR hasn't addressed incorporating something like the builder pattern yet?